src/main/java/org/fundacionjala/coding/andy/DnaStrand.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.andy;
+
+/**
+ * @author lazaro on 10/28/2019.
+ * coding.
+ */
+public final class DnaStrand {
+    private DnaStrand() { }
+
+    public static String makeComplement(final String dna) {
+        StringBuilder bld = new StringBuilder();
+        for (int i = 0; i < dna.length(); i++) {
+            switch (dna.charAt(i)) {
```

Comment:
Don't use a switch case. Consider using the replaceAll method or a HashMap.

Reply:
changed to use replace method